bin/www

```diff
@@ -12,7 +12,7 @@ var http = require('http');
  * Get port from environment and store in Express.
  */
 
-var port = normalizePort(process.env.PORT || '3001');
+var port = normalizePort(process.env.PORT || '3008');
```

Comment:
Just wondering why this was changed to 3008? Very specific number and I didn't need to change mine.

Reply:
ah no reason, I stop and restart the server many times to test and I have to change the port everytime 😅 I'll change it back